prow/config.yaml

```diff
@@ -710,6 +710,27 @@ postsubmits:
         secret:
           secretName: service-account
 
+  - name: maintenance-ci-prow-config-upload
+    branches:
+    - master
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e-prow:v20170418-c08e1094
+        args:
+        - --bare
+        - --timeout=30
+        env:
+        - name: GOOGLE_APPLICATION_CREDENTIALS
+          value: /etc/service-account/service-account.json
+        volumeMounts:
+        - name: service
+          mountPath: /etc/service-account
+          readOnly: true
+      volumes:
+      - name: service
+        secret:
+          secretName: service-account
```

Comment:
This service account won't have access to the prow cluster so it won't be able to update the cluster. That's a good thing. We'll need to add another account or do this in another way.